applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -230,25 +281,53 @@ Image transfer over SMP
 The update image can also be transferred in the background through one of the following modules:
 
 * :ref:`nrf_desktop_ble_smp` (supported only with MCUboot bootloader)
-* :ref:`nrf_desktop_dfu_mcumgr` (supported only with MCUboot bootloader)
+* :ref:`nrf_desktop_dfu_mcumgr`
   The module uses the :ref:`nrf_desktop_dfu_lock` to synchronize non-volatile memory access with other DFU methods.
   Therefore, this module should be used for configurations that enable multiple DFU transports (for example, if a configuration also enables :ref:`nrf_desktop_dfu`).
 
 The `nRF Connect Device Manager`_ application transfers the image update files over the Simple Management Protocol (SMP).
 
 To perform DFU using the `nRF Connect Device Manager`_ mobile app, complete the following steps:
 
-.. include:: /device_guides/nrf52/fota_update.rst
-   :start-after: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
-   :end-before: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
+.. tabs::
 
-.. include:: /device_guides/nrf52/fota_update.rst
-   :start-after: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
-   :end-before: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end
+   .. tab:: MCUboot
 
-.. note::
-   If the :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT` Kconfig option is enabled in the application configuration, the device rejects update image upload for the invalid slot.
-   It is recommended to enable the option if the application uses MCUboot in the direct-xip mode.
+      .. include:: /device_guides/nrf52/fota_update.rst
+         :start-after: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
+         :end-before: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
+
+      .. include:: /device_guides/nrf52/fota_update.rst
+         :start-after: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
+         :end-before: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end
+
+      .. note::
+         When the :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT` Kconfig option is enabled in the application configuration, the device rejects the update image upload for the invalid slot.
+         It is recommended to enable the option if the application uses MCUboot in the direct-xip mode.
+
+   .. tab:: SUIT
```

Comment:
Would it be possible to include these steps from the nRF54H "Working with" guide (similarly to the included content for the nRF52 Series and MCUboot)? It would be best that these steps are maintained by the Charon team as it would pretty much guarantee that they are always up to date.

Reply:
It would be a great idea, but I didn't see any fragment which describes this process in "Device Firmware Update using SUIT" in device guide for nrf54h20.

If you know where I can find it please link it here.

Reply:
Well, this is something that we should coordinate/arrange. They certainly have steps for testing the app in their sample

Reply:
With beautiful pictures:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/suit/smp_transfer/README.html#testing

Reply:
We already discussed that with Charon some time ago, here is a related ticket: https://nordicsemi.atlassian.net/browse/NCSDK-25026